Factory/Config/FilterFactoryInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Factory\Config;
+
+use Liip\ImagineBundle\Config\FilterInterface;
+
+interface FilterFactoryInterface
+{
+    /**
+     * @return string
```

Comment:
can we say that the name must match the Filter::getName that this factory handles?

Reply:
done